flotilla-server/main.go

```diff
@@ -16,6 +17,8 @@ func main() {
 			"Google Cloud project id (needed for Cloud Pub/Sub)")
 		gCloudJSONKey = flag.String("gcloud-json-key", "",
 			"Google Cloud project JSON key file (needed for Cloud Pub/Sub)")
+		coordinator = flag.String("coordinator", "", "http ip & port address for etcd")
+		flota       = flag.String("flota", "", "test group the deamon is part of")
```

Comment:
Nevermind the above two comments. Empty string can just signal to not register with etcd.